crates/matrix-sdk/src/encryption/identities/users.rs

```diff
@@ -425,7 +462,6 @@ struct OwnUserIdentity {
 struct OtherUserIdentity {
     pub(crate) inner: InnerUserIdentity,
     pub(crate) client: Client,
-    pub(crate) direct_message_room: Arc<RwLock<Option<Room>>>,
```

Comment:
Removing this and using `get_dm_room` instead would have been nice as a separate commit.

Reply:
Heh, I even forgot I did that.